packages/bpk-component-calendar/README.md

```diff
@@ -268,8 +271,40 @@ This prop determines what the colour date cell is to be displayed.
 - `CELL_TYPES.negative` - sets the calendar cell to `Panjin`
 - `CELL_TYPES.default` - sets the calendar cell to `Sky Gray Tint 02`
 
+#### `dateType` prop
+
+This property determines which selected styles will be applied to the date cell. If using ranges use `start`, `middle` and `end` to apply the correct range styles.
+
+- `DATE_TYPES.single` - sets the calendar cell to standard `Sky Blue` colour
```

Comment:
I think rather than explain the resulting style this should explain when this is used

e.g. "When the date is selected individually i.e. not as part of a range"